apps/ui/src/components/Breadcrumb.vue

```diff
@@ -4,13 +4,33 @@ import { NetworkID } from '@/types';
 const route = useRoute();
 const spacesStore = useSpacesStore();
 const proposalsStore = useProposalsStore();
-const { param } = useRouteParser('space');
+const uiStore = useUiStore();
+
+const param = ref<string>('');
+
+watchEffect(() => {
+  param.value = String(
+    route.matched[0]?.name === 'space' ? route.params.id : route.params.space
+  );
+});
+
```

Comment:
Forgot that I had this comment in mind and already submitted review.

What do you think about making it a `computed` instead of `ref` + `watchEffect`?